packages/opentelemetry-propagator-jaeger/package.json

```diff
@@ -0,0 +1,76 @@
+{
+  "name": "@opentelemetry/propagator-jaeger",
+  "version": "0.18.0",
+  "description": "OpenTelemetry Jaeger propagator provides HTTP header propagation for systems that are using Jaeger HTTP header format.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "repository": "open-telemetry/opentelemetry-js-contrib",
```

Comment:
```suggestion
  "repository": "open-telemetry/opentelemetry-js",
```

Reply:
grepped and fixed all instances of `contrib` within the new code